src/utils/parseTable.js

```diff
@@ -297,22 +304,25 @@ export function htmlToGridSettings(element, rootDocument = document) {
 
           return settings;
         }, {});
+        let cellValue = '';
 
         if (cellStyle.whiteSpace === 'nowrap') {
-          dataArr[row][col] = innerHTML.replace(/[\r\n][\x20]{0,2}/gim, '\x20')
+          cellValue = innerHTML.replace(/[\r\n][\x20]{0,2}/gim, '\x20')
             .replace(/<br(\s*|\/)>/gim, '\r\n')
-            .replace(/(<([^>]+)>)/gi, '')
-            .replace(/&nbsp;/gi, '\x20');
+            .replace(regSpace, '\x20');
 
         } else if (generator && /excel/gi.test(generator.content)) {
-          dataArr[row][col] = innerHTML.replace(/[\r\n][\x20]{0,2}/g, '\x20')
+          cellValue = innerHTML.replace(/[\r\n][\x20]{0,2}/g, '\x20')
             .replace(/<br(\s*|\/)>[\r\n]?[\x20]{0,3}/gim, '\r\n')
-            .replace(/(<([^>]+)>)/gi, '')
-            .replace(/&nbsp;/gi, '\x20');
+            .replace(regSpace, '\x20');
```

Comment:
Why isn't `regSpace` a part of the `ESCAPED_HTML_CHARS`? Looks like we change it in every case anyway.

Reply:
Good point. Changed in https://github.com/handsontable/handsontable/pull/6424/commits/dab492687ff43fc45e0bb9a7104550b99d1a8dbb